roles/control_center/tasks/main.yml

```
@@ -268,6 +268,11 @@
   notify: restart control center
   when: certs_updated|bool
 
+- name: Verify force Restart
+  command: /bin/true
+  notify: restart control center
+  when: confluent_force_service_restart|bool
```

We need variables for each component, as thats a standard

```suggestion
- name: Force Restart
  command: /bin/true
  notify: restart control center
  when: control_center_force_restart|bool
  tags: restart
```

```
- name: Health Check
  include_tasks: health_check.yml
  when:
    - control_center_health_checks_enabled|bool
    - not ansible_check_mode
  tags: restart
```

And maybe you want to use tags as well, so this could be orchestrated with 
```
--tags control_center,restart
```